HW_9/get_csv.py

```diff
@@ -0,0 +1,18 @@
+from random_csv import csv_rand
+import csv
+
+
+def get_num_csv(func):
+    lst = []
+    csv_rand(100, 1, 100)
+
+    def wrapper(*args, **kwargs):
+        with open(f'{csv_rand.__name__}.csv', 'r', newline='') as f:
+            reader = csv.reader(f)
+            for i in reader:
+                args = map(int, i)
+                lst.append(func(*args, **kwargs))
+            return lst
```

Comment:
Помните, я говорил на семинаре, что функция обертка должна как бы "мимикрировать" под оригинальную функцию, то есть она должна принимать и возвращать то же самое, что и декорируемая функция. Но у вас обертка принимает и возвращает список. Зачем?
Кроме того, при написании декораторов следует придерживаться принципа, что ни о каком другом декораторе, который тоже будет применяться к функции, неизвестно. Но ваш декоратор знает, что от функции придет список
```suggestion
    csv_rand(100, 1, 100)
    def wrapper(*args, **kwargs):
        with open(f'{csv_rand.__name__}.csv', 'r', newline='') as f:
            reader = csv.reader(f)
            for i in reader:
                args = 
                func(*map(int, i))
```